crates/anvil-polkadot/src/genesis.rs

```diff
@@ -0,0 +1,34 @@
+//! Genesis settings
+
+use crate::config::AnvilNodeConfig;
+use alloy_genesis::GenesisAccount;
+use alloy_primitives::{Address, U256};
+use foundry_evm::revm::primitives::AccountInfo;
+use std::collections::BTreeMap;
+
+/// Genesis settings
+#[derive(Clone, Debug, Default)]
+pub struct GenesisConfig {
```

Comment:
Maybe worth adding the base_fee_per_gas [1], gas_limit  [2] and chain_id [3] as well since we know we want to modify them next. Can be added in a follow up too. Other than this, I would say that this genesis config struct looks pretty much ready from a mandatory config standpoint after a quick glance at the genesis config from alloy-genesis.

[1] https://docs.rs/alloy/latest/alloy/genesis/struct.Genesis.html#structfield.base_fee_per_gas
[2] https://docs.rs/alloy/latest/alloy/genesis/struct.Genesis.html#structfield.gas_limit
[3] https://docs.rs/alloy/latest/alloy/genesis/struct.ChainConfig.html#structfield.chain_id